assets/src/dashboard/components/storyAnimation/provider.js

```diff
@@ -114,7 +115,16 @@ function Provider({ animations, children, onWAAPIFinish }) {
 
   const WAAPIAnimationMethods = useMemo(() => {
     const play = () =>
-      WAAPIAnimations.forEach((animation) => animation?.play());
+      WAAPIAnimations.forEach((animation) => {
```

Comment:
Console log this `animation` object if you want to see the object with the `_holdTime` prop.